### PR TITLE
Feature/F01 Prepare main page skeleton

### PR DESCRIPTION
**Pull request by DVZakharov**
added navigation menu,
a place has been allocated for a carousel of wallpapers and news

### CODE REVIEW

**Review by andreygolubkow — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -1,18 +1,83 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/App.tsx]: +

File: src/App.tsx [hunk 2]
```diff
@@ -1,18 +1,83 @@
...
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/App.tsx]: +

File: src/App.tsx [hunk 3]
```diff
@@ -1,18 +1,83 @@
...
+        textDecoration: 'none',
+        bg: useColorModeValue('gray.200', 'gray.700'),
+      }}
+      href={`#${props.children}`}>
+      {props.children}
+    </Box>
+  );
+};
+
+export default function MainMenu() {
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/App.tsx]: +

File: src/App.tsx [hunk 4]
```diff
@@ -1,18 +1,83 @@
...
+import Footer from './Footer';
 
-function App() {
+const News = lazy(() => import( './News' ));
+
+interface Props {
+  children: string;
+}
+
+const NavLink = (props: Props) => {
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/App.tsx]: +

File: src/App.tsx [hunk 5]
```diff
@@ -1,18 +1,83 @@
...
   return (
     <>
-      <div>
-        <a href="https://vk.com/takt_tusur" target="_blank">
-          <img src={taktLogo} className="logo" alt="Перейти на страницу в VK" />
-        </a>
-      </div>
-      <h1>Туристско-альпинистский клуб ТУСУРа</h1>
-      <p>Скоро тут будет сайт</p>
+      <Box w='100%' position='fixed' bg={useColorModeValue('gray.100', 'gray.900')} px={4}>
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/App.tsx]: +

File: src/CurrentEvents.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/CurrentEvents.tsx]: +

File: src/Equipment.tsx [hunk 7]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/Equipment.tsx]: +

File: src/Footer.tsx [hunk 8]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/Footer.tsx]: +

File: src/News.tsx [hunk 9]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/News.tsx]: +

File: src/WallpaperCarousel.tsx [hunk 10]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/WallpaperCarousel.tsx]: +

File: src/main.tsx [hunk 11]
```diff
@@ -1,10 +1,14 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
[the remark this answers is not in the document]
Comment (DVZakharov) [reply src/main.tsx]: +

**Review by DVZakharov — COMMENTED** [review]
+

**Review by andreygolubkow — APPROVED** [review]

**Pull request merged**